ext/flash/01_http.js

```diff
@@ -188,6 +188,44 @@
     return hostname === "0.0.0.0" ? "localhost" : hostname;
   }
 
+  function writeFixedResponse(
+    server,
+    requestId,
+    response,
+    end,
+    respondFast,
+  ) {
+    let nwritten = 0;
+    // TypedArray
+    if (typeof response !== "string") {
+      nwritten = respondFast(requestId, response, end);
+    } else {
+      // string
+      const maybeResponse = stringResources[response];
```

Comment:
i think we should remove this response cache - it seems to cause a large amount of memory usage and GC when responses vary many times within a second (e.g. with a random number in each response). it is also happening without knowledge of api consumer and might be better as a user configurable option in a future release?

if i understand correctly, it also skips the fast-api-call op when using the cached version?

Reply:
It takes the fast api path for the cached version. The cache is essentially a `Record<string, Uint8Array>`. Let's address this in a follow up